File: tests/src/main.server.lua
-- Compiled with roblox-ts v1.1.1
local TS = require(game:GetService("ReplicatedStorage"):WaitForChild("include"):WaitForChild("RuntimeLib"))
local TestEZ = TS.import(script, TS.getModule(script, "testez").src)
local results = TestEZ.TestBootstrap:run({ game:GetService("ServerScriptService").tests })
if #results.errors > 0 or results.failureCount > 0 then
	error("Tests failed!")
end

File: tests/src/main.server.ts
import type TestEZType from "@rbxts/testez";
const TestEZ = require(game.GetService("ReplicatedStorage").include.node_modules["@rbxts"].testez.src) as TestEZType;
const results = TestEZ.TestBootstrap.run([game.GetService("ServerScriptService").tests]);
if (results.errors.size() > 0 || results.failureCount > 0) {
	error("Tests failed!");
}
